.ci/stack_build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -xeo pipefail
+
+apt install wget -y
+wget -qO- https://get.haskellstack.org/ | sh
```

Comment:
I know this is the "official way", but `wget ... | sh` just feel so wrong to me.